workspace/__ardulib__/SSD1306/SSD1306.cpp

```diff
@@ -0,0 +1,90 @@
+
+#include "SSD1306.h"
+
+SSD1306::SSD1306(TwoWire* wire, uint8_t i2cAddress) {
+    _wire = wire ? wire : &Wire;
+    _i2cAddress = i2cAddress;
+    _width = SSD1306_DISPLAY_WIDTH;
+    _height = SSD1306_DISPLAY_HEIGHT;
+}
+
+void SSD1306::sendCommand(uint8_t command) {
+    _wire->beginTransmission(_i2cAddress);
+    _wire->write(0x80);
+    _wire->write(command);
+    _wire->endTransmission();
+}
+
+void SSD1306::setBuffer(uint8_t* buffer) {
+    _buffer = buffer;
```

Comment:
Does it make sense to keep a dedicated method? One can forget to call it or change in the middle of the object lifetime, is it expected. I’d pass `buffer` to the constructor.